src/satellite_images_nso_datascience/annotations/data_preparation.py

```diff
@@ -13,16 +13,35 @@ def get_flattened_pixels_for_polygon(
     """
     Cuts polygon out of dataset and flattens the bands of dataset into a single pandas DataFrame
     """
-    cropped_to_polygon, _ = mask(dataset, [polygon], crop=True)
+    cropped_to_polygon, out_transform = mask(dataset, [polygon], crop=True)
 
-    bands = dataset.count
     column_names = dataset.descriptions
 
-    df = pd.DataFrame(
-        cropped_to_polygon.reshape(bands, -1).transpose(),
-        columns=column_names,
-        dtype=float,
+    z_shape = cropped_to_polygon.shape[0]
```

Comment:
What has changed functionally in this code? What is the motivation to make this change? I'm missing something obvious I think.

Reply:
Okay, then change it back? What do you mean I don't remember these changes?

Reply:
i remember that the reason for this change was to also include x and y coordinates when extracting pixels values from a .tif file and not doing it somewhere differently

Reply:
which would further be used for making hexagons, so definitly these changes are wanted

Reply:
Clear!